docs/features/shared-access-key.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Authentication with shared access keys via ASP.NET Core authentication filters"
+layout: default
+---
+
+The `Arcus.WebApi.Security` package provides an mechanism that uses shared access keys to authenticate users.
+This authentication process consists of two parts:
+
+1. Looks for the configured HTTP request header that contains the shared access key
+2. Matches this value with the **Key Vault** secret that is requested via a `ICachedSecretProvider` or `ISecretProvider`
```

Comment:
It still mentions Key Vault which is not the case, maybe they wrote there own.

I propose the following:
```suggestion
2. Shared access key matches the value with the secret stored, determined via configured secret provider
```